CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ if (PERFSUITE_ENABLE_WARNINGS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror")
 endif()
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mavx2")
```

Comment:
This flag should go in the scripts/lc-builds/XXX files.  Also I think there is a architecture agnostic flag for (at least gnu and clang) like -march=native or something... in case the machine has SSE, AVX, AVX2 or AVX512, it will pick the best one.

Reply:
Yes, I saw the lc-builds file and this is where I have it now. Not sure how this slipped in..but thanks for the feedback!